### ISSUE TITLE
Phone number

### ISSUE THREAD

**Issue by RhysMcCabe** [kelvinf96/PasswordManagerClientApp#25]
@kelvinf96

Can / should we remove the phone number step?

Its not being stored or sent to the API.

We either:
1. remove it from the frontend.
2. send it to the API to store it on the user model. Every request checks the androidID AND phone number.

The second one is a lot more work, I would like to do it, but I think at this stage we should not be adding anymore features.

What do you think?

**Comment by kelvinf96** [comment]
Yeah its definitely pointless at the minute, I would think for this project using the android Id alone should be okay as its to show we can create an api and client to make requests so security isn't really top priority. It would be nice to have it check both the phone number and ensure the android id is matching for  that number, but think we might be better off spending the time on some testing
